docs/reference/contrib/settings.md

```diff
@@ -0,0 +1,379 @@
+# Settings
+
+The `settings` module allows you to:
```

Comment:
I like the completeness of this, though am unsure about:

1. using the word "global" - recent feedback was to rename this work to "generic settings" instead of "global settings", as "global" may confuse things if we're working with a multilingual setup.
2. removing the two bullet points that currently follow this line; I think it's useful to clearly call out that there are two separate workflows.

Thoughts?